.github/workflows/deploy-cluster.yml

```diff
@@ -62,6 +62,17 @@ jobs:
         terraform_root_folder: templates/new_service/terraform/domains/environment_domains
         terrafile_environment: development
 
+    - name: Send Slack notification on failure
+      if: failure() && github.ref == 'refs/heads/main'
+      uses: rtCamp/action-slack-notify@master
+      env:
+        SLACK_COLOR: ${{env.SLACK_ERROR}}
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_TITLE: A Cluster update failure has occurred
+        SLACK_MESSAGE: |
+          The terraform validation during cluster update to ${{ matrix.environment }} has failed
```

Comment:
This job doesn't have a matrix so `${{ matrix.environment }}` would return nothing